terraform/github/data-catalogue.tf

```diff
@@ -16,23 +16,43 @@ module "data_catalogue_team" {
   ]
 }
 
-module "data_catalogue_repository" {
+# module "data_catalogue_parent_repository" {
```

Comment:
Will add this in a follow up PR - not sure how the provider will handle it if I apply the move & recreate at the same time